superset/config.py

```diff
@@ -1102,6 +1103,13 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
     return msg
 
 
+# For database connections, this dictionary will remove engines from the available
+# list/dropdown if you do not want these dbs to show as available.
+# The available list is generated by driver installed, and some drivers have multiple
+# engines.
+# e.g., DBS_AVAILABLE_BLACKLIST: Dict[str, Tuple[str, ...]] = {"databricks": ("pyhive", "pyodbc")}
+DBS_AVAILABLE_BLACKLIST: Dict[str, Tuple[str, ...]] = {}
```

Comment:
A few years ago I replaced white-/blacklist terminology with allow-/denylist: #10320, it would be nice to try to use the more neutral terminology where possible

Reply:
That's a great idea.. I'll implement that, thanks!